homeassistant/components/mqtt/climate.py

```diff
@@ -336,15 +297,6 @@ def valid_preset_mode_configuration(config):
     cv.PLATFORM_SCHEMA.extend(_PLATFORM_SCHEMA_BASE.schema),
     # CONF_SEND_IF_OFF is deprecated, support will be removed with release 2022.9
     cv.deprecated(CONF_SEND_IF_OFF),
-    # AWAY and HOLD mode topics and templates are deprecated, support will be removed with release 2022.9
-    cv.deprecated(CONF_AWAY_MODE_COMMAND_TOPIC),
-    cv.deprecated(CONF_AWAY_MODE_STATE_TEMPLATE),
-    cv.deprecated(CONF_AWAY_MODE_STATE_TOPIC),
-    cv.deprecated(CONF_HOLD_COMMAND_TEMPLATE),
-    cv.deprecated(CONF_HOLD_COMMAND_TOPIC),
-    cv.deprecated(CONF_HOLD_STATE_TEMPLATE),
-    cv.deprecated(CONF_HOLD_STATE_TOPIC),
-    cv.deprecated(CONF_HOLD_LIST),
```

Comment:
Shouldn't we use `cv.removed` here too?

Reply:
True already saw it myself, I fixed it when rebasing